ci/gpu/build.sh

```diff
@@ -83,7 +83,8 @@ gpuci_conda_retry install -y \
 
 # https://docs.rapids.ai/maintainers/depmgmt/
 # gpuci_conda_retry remove --force rapids-build-env rapids-notebook-env
-# gpuci_conda_retry install -y "your-pkg=1.0.0"
+gpuci_conda_retry install -y "numpy<1.20"
```

Comment:
this will need to be removed before the PR is merged. requesting changes here to prevent @kkraus14's `@gpucibot merge` comment from automatically merging this PR.

Reply:
@ajschmidt8 this is temporary to unblock CI as there's a bug in upstream Dask DataFrame related to numpy 1.20. We plan to fix that upstream and then remove this in a future PR.

Other libraries like cuML really want numpy 1.20 as it fixes other bugs for them so we don't want to pin `integration`.